app/packages/state/src/recoil/groups.ts

```diff
@@ -334,16 +334,10 @@ export const activeModalSample = selector({
   },
 });
 
-export const groupStatistics = graphQLSyncFragmentAtomFamily<
-  datasetFragment$key,
-  "group" | "slice",
-  boolean
->(
-  { fragments: [datasetFragment], keys: ["dataset"], default: "slice" },
-  {
-    key: "groupStatistics",
-  }
-);
+export const groupStatisticsState = atom<"group" | "slice">({
+  key: "groupStatisticsState",
+  default: "slice",
+});
```

Comment:
This should be reverted to original boolean a atom family, or else toggling the value in the modal will cause unwanted grid updates
```suggestion
export const groupStatistics = atomFamily<"group" | "slice">({
  key: "groupStatistics",
  default: "slice",
});
```